Automattic/sensei
# Handle activating / installing WooCommerce from the Extensions page

Based on https://github.com/Automattic/sensei/pull/4213.

### Changes proposed in this Pull Request
Display a notice if there are updates waiting that are delivered via WooCommerce.com, but WooCommerce is not installed or activated.

### Testing instructions
- Uninstall WooCommerce.
- Install an old version of Content Drip.
- Go to the Extensions page and ensure the following notice is displayed:

![Screen Shot 2021-05-04 at 3 20 39 PM](https://user-images.githubusercontent.com/1190420/117058476-a971d100-acec-11eb-9d4f-31dc7437467e.jpg)
- Click on _Install WooCommerce_ and ensure you're taken to the Plugins page with WooCommerce displayed as the first listing.
- Install WooCommerce but don't activate it.
- Go to the Extensions page and ensure the following notice is displayed:

![Screen Shot 2021-05-04 at 3 19 14 PM](https://user-images.githubusercontent.com/1190420/117058597-d7efac00-acec-11eb-8287-1e0794221f45.jpg)
- Click on _Activate WooCommerce_ and ensure WooCommerce is successfully activated.
- Disconnect your WooCommerce.com account if it's connected.
- Go to the Extensions page and ensure the following notice is displayed:

![Screen Shot 2021-05-04 at 3 19 48 PM](https://user-images.githubusercontent.com/1190420/117058762-01a8d300-aced-11eb-909f-55f7c995b634.jpg)
- Click the _Connect account_ button and ensure you are taken through the WooCommerce connection flow.
- Once connected, revisit the Extensions page and ensure the Content Drip update is now displayed as part of the _Update all_ notice.
- Bonus: Install an old version of WooCommerce Paid Courses and repeat the above steps.
